client/apps/edit/components/admin/index.coffee

```diff
@@ -145,6 +147,36 @@ module.exports = class EditAdmin extends Backbone.View
     else
       select.setState loading: false
 
+  setupContributingAuthors: ->
+    AutocompleteList = require '../../../../components/autocomplete_list/index.coffee'
+    @contributing_authors = @article.get 'contributing_authors' or []
+    list = new AutocompleteList @$('#edit-admin-contributing-authors .admin-form-right')[0],
+      name: 'contributing_authors[]'
+      url: "#{sd.ARTSY_URL}/api/v1/match/users?term=%QUERY"
+      placeholder: 'Search by user name or email...'
+      filter: (users) -> for user in users
+        { id: user.id, value: _.compact([user.name, user.email]).join(', ') }
+      selected: (e, item, items) =>
+        @article.save contributing_authors: _.pluck items, 'id'
+      removed: (e, item, items) =>
+        @article.save contributing_authors: _.without(_.pluck(items, 'id'),item.id)
+    if ids = @article.get 'contributing_authors'
+      @authors = []
+      async.each ids, (id, cb) =>
```

Comment:
Nice! Looks like you grasped async pretty easily! Let me know if anything is unclear with how that control flow lib works.